lib/auth/auth_with_roles.go

```diff
@@ -883,16 +883,19 @@ func (a *ServerWithRoles) ListResources(ctx context.Context, req proto.ListResou
 	}
 
 	limit := int(req.Limit)
+	actionVerbs := []string{types.VerbList, types.VerbRead}
+
 	switch req.ResourceType {
-	case types.KindDatabaseServer,
-		types.KindAppServer,
-		types.KindNode,
-		types.KindKubeService:
+	case types.KindNode:
+		actionVerbs = []string{types.VerbList}
```

Comment:
Why do nodes only need "list" but others need both "list" and "read"?

Reply:
I don't know... I was advised to keep it as is since it's been like that for a while (GetNodes that is). I tried to hunt it down. [This](https://github.com/gravitational/teleport/blob/8838a419aff8b1f1e83be01294c68eb083937767/lib/auth/auth_with_roles.go#L194) is when it got added, then it magically disappears, I can't find the PR that removes it...

Reply:
@kimlisa Strange that that PR touched nodes ACL, it doesn't seem related at all to the actual thing the PR implemented. Should we just keep VerbList permission for other agents too then for consistency?